torch/_inductor/sizevars.py

```diff
@@ -334,6 +334,9 @@ def guard_static_shape(self, left: Expr) -> int:
         self.guard_equals(left, sympy.Integer(right))
         return int(right)
 
+    def guard_static_shapes(self, left: List[Expr]) -> List[int]:
```

Comment:
this doesn't seem to be used in this PR, is it for the following PRs in the stack?

Reply:
Yeah, it is used by the convolution rewrite down the stack.